backend/src/Designer/EventHandlers/LanguageTextsKeyChanged/LanguageTextsIdChanged.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Altinn.Studio.Designer.Events;
+using Altinn.Studio.Designer.Hubs.SyncHub;
+using Altinn.Studio.Designer.Services.Interfaces;
+using MediatR;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Altinn.Studio.Designer.EventHandlers.LanguageTextsKeyChanged;
```

Comment:
I do like the suggested name better, but there is one issue; not only layout files are changed by this. It might also change options files. And probably more files later. So i've changed it to `TextIdChangedHandler`, is this alright?

Reply:
I think we should have two separate files for it. That is at least the way it has been done for the other cases such as componentID and TaskId 😊 
`TextIdChangedLayoutsHandler` and `TextIdChangedOptionsHandler`